instrumentation/rabbitmq-2.7/javaagent/src/test/groovy/WithRabbitMqTrait.groovy

```diff
@@ -28,4 +28,13 @@ trait WithRabbitMqTrait {
   def stopRabbit() {
     rabbitMqContainer?.stop()
   }
+
+  static boolean isContainerIpAddress(Object ip) {
+    def containerIp = rabbitMqContainer.containerIpAddress
+    // getContainerIpAddress() can return "localhost", which obviously is not an IP address
```

Comment:
Should we report this as bug to Testcontainers?

Reply:
https://github.com/testcontainers/testcontainers-java/issues/4715

Reply:
Turns out this method is supposed to work this way and will be deprecated 😅 
I changed the assertion slightly so that it matches any IP address.